frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.module.css

```diff
@@ -46,13 +55,23 @@
   align-self: left;
 }
 
+.presentShoutoutFrom {
+  margin-top: 0 !important;
+}
+
 .shoutoutHide {
   display: flex;
   justify-content: space-between;
   margin-top: 0 !important;
   margin-bottom: 0 !important;
 }
 
+.presentShoutoutMessage {
+  color: black;
+  margin-top: 0 !important;
+  font-size: 25px !important;
+}
+
```

Comment:
I think maybe a good first "refactoring" or "cleaniup" PR next semester would be to look into why we need so many `important` tags for this frontend. Hopefully we should be able to figure out a way to avoid this, since like Henry said earlier its not really good practice to have so many of these.